### PR TITLE
Feat/ convertire et exporter les commandes

### PR DESCRIPTION
**Pull request by toufik1978-del**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -159,12 +116,85 @@ client.on('error', (err) => {
...
+} catch (err) {
+if (err.message.includes('Aucun en-tête END trouvé')) {
+    console.error('Erreur : Format ZIP invalide ou non pris en charge. Le fichier est peut-être corrompu.');
+  } else {
+    console.error('Erreur inattendue lors de lextraction ZIP :', err);
+  }
+} 
+
+// FTP - Importation  des commandes
+const Order = require('./models/orders'); //
```
Comment (WilliamZimmermann) [inline app.js:134]: Why here your const starts with uppercase and then, in line 138, with lower case? You should define a standard. I suggest always lowercase for vars or constants used in only one file. Classes/Objects usually follow the CamelCase standard.

File: app.js [hunk 2]
```diff
@@ -159,12 +116,85 @@ client.on('error', (err) => {
...
+    // Créer le fichier avec le contenu spécifié
+    fs.writeFileSync(filePath, fileContent);
+
+    console.log(`Le fichier ${fileName} a été créé avec succès dans le dossier 'orders'.`);
+  } catch (error) {
+    console.error('Une erreur est survenue lors de la création du fichier :', error);
+  }
+};
+// Appeler la fonction pour créer le fichier dans le dossier 'orders'
+createFileInOrdersFolder();
```
Comment (WilliamZimmermann) [inline app.js:173]: Your file should be better organized and "ordered". Please, use Prettier or other library. Or right click with VSCode -> Format.

Comment (toufik1978-del) [reply app.js]: C'est fait

File: app.js (hunk 1 above)
Comment (toufik1978-del) [reply app.js]: C'est corrigé

**Review by jpjutras82 — APPROVED** [review]
Beau code!
Fantastique!

**Pull request merged**